components/post_view/post_info/index.ts

```diff
@@ -2,22 +2,24 @@
 // See LICENSE.txt for license information.
 
 import {connect} from 'react-redux';
-import {bindActionCreators} from 'redux';
+import {AnyAction, bindActionCreators, Dispatch} from 'redux';
 import {removePost} from 'mattermost-redux/actions/posts';
 import {isCurrentChannelReadOnly} from 'mattermost-redux/selectors/entities/channels';
 import {getCurrentTeamId} from 'mattermost-redux/selectors/entities/teams';
 import {get} from 'mattermost-redux/selectors/entities/preferences';
 import {getConfig} from 'mattermost-redux/selectors/entities/general';
 
+import {GlobalState} from 'types/store';
+
 import {emitShortcutReactToLastPostFrom} from 'actions/post_actions.jsx';
 import {Preferences} from 'utils/constants';
 import * as PostUtils from 'utils/post_utils.jsx';
 import {getSelectedPostCard} from 'selectors/rhs';
 import {getShortcutReactToLastPostEmittedFrom} from 'selectors/emojis';
 
-import PostInfo from './post_info.jsx';
+import PostInfo, {Props} from './post_info';
 
-function mapStateToProps(state, ownProps) {
+function mapStateToProps(state: GlobalState, ownProps: Props) {
```

Comment:
@vraravam So I pulled down your branch and found the issue. It appears that by importing `Props` from your component, that the redux-connected component is expecting that you pass it all of the props that are expected into the redux-connected component, which would include everything required by the component that's provided by the redux connector.

You'll want to define an `OwnProps` type that just contains the props that are used by `mapStateToProps` and use that for the type argument here instead of importing from your actual component.

```suggestion
type OwnProps = {
    post: Post;
}

function mapStateToProps(state: GlobalState, ownProps: OwnProps) {
```

Reply:
thansk @devinbinnie - i will make the change and commit in a few mins.

Reply:
done - that fixed the issues! thanks once again!